examples/mobile/UIComponents/components/dialogs/pickers/calendar.html

```diff
@@ -18,7 +18,9 @@ <h1>
 		</div>
 		<div class="ui-content">
 			<ul class="ui-listview ui-content-area">
-				<div class="ui-calendar" data-past-selection="true"></div>
+				<div class="ui-calendar-container">
```

Comment:
Why are you adding the "ui-calendar-container""?
You can apply the style without container.

Reply:
I removed div.
and add style to padding in ui-content-area.